src/VisualStudio/Core/Impl/Options/GridOptionPreviewControl.xaml.cs

```diff
@@ -29,7 +30,7 @@ private void Options_SelectionChanged(object sender, SelectionChangedEventArgs e
             var dataGrid = (DataGrid)sender;
             var codeStyleItem = (AbstractCodeStyleOptionViewModel)dataGrid.SelectedItem;
 
-            if (codeStyleItem != null && codeStyleItem.NotificationsAvailable)
+            if (codeStyleItem != null)
```

Comment:
yep, that bad. it shouldn't have been checked in in the first place. I was testing something locally and unintentionally shipped it in the preview, the resulting bug is that if severity levels weren't available, previews will not be updated if user clicks on a different code style in the grid.